src/mscorlib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -123,13 +129,17 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
         {
             EnumCalendarsData data = new EnumCalendarsData();
             data.userOverride = 0;
+#if INSIDE_CLR
```

Comment:
I didn't introduce INSIDE_CLR, it was there and I just used it. I can use CORECLR but we need to check if this not compiled for corert/PN